src/__tests__/core/forgotPassword.test.ts

```
@@ -0,0 +1,432 @@
+      const passwordMatch = logCall[0].match(/: (.+)$/);
+      const tempPassword = passwordMatch ? passwordMatch[1] : '';
+      
+      expect(tempPassword).toHaveLength(20);
+    });
+  });
+
+  describe('Error handling', () => {
+    it('should handle database adapter without updateUser method', async () => {
+      // Mock a limited adapter by setting updateUser to undefined
+      const originalUpdateUser = mockAdapter.updateUser;
+      delete mockAdapter.updateUser;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)